PostFeedbackHttpTrigger/tests/fixtures/valid_feedback.json

```diff
@@ -0,0 +1,16 @@
+{
+    "page": "course finder",
+    "is_useful": true,
+    "questions": [{
+            "title": "postcodes are great",
+            "feedback": ""
+        },
+        {
+            "title": "you need a favicon",
+            "feedback": "see the title"
+
```

Comment:
I would tend to keep things a little tighter, no need for empty lines when the file is a json document

Reply:
👍